10-inference.Rmd

```diff
@@ -0,0 +1,789 @@
+        axis.text.x = element_blank(), 
+        axis.title.y =element_blank(), 
+        axis.ticks.y = element_blank(),
+        axis.text.y = element_blank()) 
+
+mygb = function(x,y) {
+  grid.bezier(x=x, y=y, gp=gpar(fill="black"), 
+              arrow=arrow(type="closed", length=unit(2,"mm")))}
+
+grid.arrange(pop_dist, 
+             sam_dist, 
+             boot1_dist, 
+             boot2_dist, 
+             boot3_dist,
+             bootstrap_dist,
+  widths = c(1, 1, 1, 1, 1),
+  layout_matrix = rbind(c(1, NA, 3, NA, 6),
+                        c(1, 2, 4, NA, 6),
+                        c(NA, NA, 5, NA, NA)
+)) 
+grid.text(paste("mean =", round(mean(boot1$price)), sep = " "), x = unit(0.7, "npc"), 
+          y = unit(.8, "npc"),gp = gpar(fontsize=12))
+grid.text(paste("mean =", round(mean(boot2$price)), sep = " "), x = unit(0.7, "npc"), 
+          y = unit(.5, "npc"),gp = gpar(fontsize=12))
+grid.text(paste("mean =", round(mean(boot3$price)), sep = " "), x = unit(0.7, "npc"), 
+          y = unit(.2, "npc"),gp = gpar(fontsize=12))
+# arrows: pop to sample 
+mygb(x=c(0.15,0.3,0.3,0.3), y=c(0.8,0.8,0.65,0.65))
+
+# arrows: sample to resamples
+mygb(x=c(0.35,0.35,0.42,0.42), y=c(0.55,0.55,0.8,0.8))
+mygb(x=c(0.35,0.35,0.42,0.42), y=c(0.5,0.5,0.5,0.5))
+mygb(x=c(0.35,0.35,0.42,0.42), y=c(0.45,0.45,0.2,0.2))
+
+# arrows: resamples to statistics
+mygb(x=c(0.57,0.57,0.62,0.62), y=c(0.8,0.8,0.8,0.8))
+mygb(x=c(0.57,0.57,0.62,0.62), y=c(0.5,0.5,0.5,0.5))
+mygb(x=c(0.57,0.57,0.62,0.62), y=c(0.2,0.2,0.2,0.2))
+
+# arrows: statistics to boot
+mygb(x=c(0.77,0.77,0.83,0.83), y=c(0.8,0.8,0.7,0.7))
+mygb(x=c(0.77,0.77,0.83,0.83), y=c(0.5,0.5,0.6,0.6))
+mygb(x=c(0.77,0.77,0.83,0.83), y=c(0.2,0.2,0.4,0.4))
+
+# dots 
+grid.text(".", x = unit(0.45, "npc"), 
+          y = unit(0.04, "npc"),gp = gpar(fontsize=15))
+grid.text(".", x = unit(0.45, "npc"), 
+          y = unit(0.06, "npc"),gp = gpar(fontsize=15))
+grid.text(".", x = unit(0.45, "npc"), 
+          y = unit(0.025, "npc"),gp = gpar(fontsize=15))
+
+grid.text(".", x = unit(0.7, "npc"), 
+          y = unit(0.14, "npc"),gp = gpar(fontsize=15))
+grid.text(".", x = unit(0.7, "npc"), 
+          y = unit(0.16, "npc"),gp = gpar(fontsize=15))
+grid.text(".", x = unit(0.7, "npc"), 
+          y = unit(0.18, "npc"),gp = gpar(fontsize=15))
+
+```
+
+### Using the bootstrap to calculate a plausible range  
+
+Now that we have constructed our bootstrap distribution let's use it to create
+an approximate bootstrap confidence interval, a range of plausible values for the population mean. We will construct a 95\% percentile bootstrap confidence interval and find the range of values that cover the middle 95\% of the bootstrap distribution. Note that there's nothing particularly special about 95\%, we could have used other confidence levels, such as 90\% or 99\%. The confidence level you choose will depend on your application, which is beyond the scope of this textbook.
```

Comment:
We need to define a 95% ci here, or this will be jibberish to the students. 

In my MDS notes, I say this (this probably comes from Modern Dive originally, and so we need to be careful to make this our own):

What does a 95% confidence interval mean?

It means that if we were to repeat the process of sampling and calculating a 95% confidence interval many many times, 95% of the time we would expect our population parameter’s value to lie within the confidence interval.

Why 95%?

Commonly used confidence levels include 90%, 95%, and 99%. All other things being equal, higher confidence levels correspond to wider confidence intervals and lower confidence levels correspond to narrower confidence intervals. The level chosen depends on the downstream analysis and what chance you are willing to take of being wrong.

Verbally then I also add that if you have a downstream decision/application where the cost of missing the population parameter in you confidence intervals is high (e.g., impacts human health or well being) then you would tend to choose a higher confidence level.